bundles/org.dataflowanalysis.analysis.dfd/src/org/dataflowanalysis/analysis/dfd/core/DFDVertex.java

```diff
@@ -111,7 +111,7 @@ private void evaluatePreviousVertices() {
      */
     private void fillMapOfIncomingLabelsPerPin(Pin pin, Map<Pin, List<Label>> inputPinsIncomingLabelMap) {
     	for (var previousVertex : this.getPinDFDVertexMap().values()) {
-            for (var dfv : previousVertex.getAllOutgoingDataFlowVariables()) {
+            for (var dfv : previousVertex.getAllOutgoingDataCharacteristics()) {
```

Comment:
dfv means DataFlowVariable? Should be changed to avoid confusion.

Reply:
Fixed in 3f60f108287df2be592d00f602cab4125a61c1f6